go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -57,11 +57,12 @@ func NewController(client clientset.Interface,
 	klog.V(4).Info("Creating event broadcaster")
 
 	c := &Controller{
-		client:           client,
-		nbClient:         nbClient,
-		queue:            workqueue.NewNamedRateLimitingQueue(newRatelimiter(100), controllerName),
-		workerLoopPeriod: time.Second,
-		alreadyApplied:   map[string][]LB{},
+		client:                  client,
+		nbClient:                nbClient,
+		queue:                   workqueue.NewNamedRateLimitingQueue(newRatelimiter(100), controllerName),
+		workerLoopPeriod:        time.Second,
+		alreadyApplied:          map[string][]LB{},
+		alreadyAppliedTemplates: TemplateMap{},
```

Comment:
`alreadyAppliedTemplates` looks like it is unused?

Reply:
Oops, left-over.  I'll remove it.